apiary.apib

```diff
@@ -846,6 +846,60 @@ This operations is available only for organization administrators.
                 "sharing": "organization-project"
             }
 
+### Share Bucket To Users [POST /v2/storage/buckets/{bucket_id}/share-to-users]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** for this bucket to a specific set of users in the organization.
+Any user who is a member of one of the targeted users will be able to link to this bucket.
```

Comment:
Any user who is included in the targetUsers array will be able to link to this bucket.